test/__snapshots__/index.test.ts.snap

```diff
@@ -2114,10 +2114,12 @@ Exhibit B - “Incompatible With Secondary Licenses” Notice
       the Mozilla Public License, v. 2.0.
 ",
   "README.md": "
-# Terraform CDK random Provider undefined
+# Terraform CDK random Provider tracks undefined
```

Comment:
Is undefined expected here for the test? 🤔 Might be correct, as it's not initialized

Reply:
Yea the provider version comes from splitting the provider name on `@`. In the test we are only passing `random` so it is undefined. Can't remember why that was the case, but I'll try changing it to include a version.

Reply:
Just updated the test, and is looking good 👍. Snapshots look more realistic 😁